kube-dns failed to start

kube-dns on master branch failed to start:

```sh
# kubectl logs kube-dns-1476438210-rzf6r -c kubedns
...
I0814 09:53:28.164431     104 server.go:130] Setting up Healthz Handler (/readiness)
I0814 09:53:28.164449     104 server.go:135] Setting up cache handler (/cache)
I0814 09:53:28.164455     104 server.go:146] Setting up pprof handler (/debug/pprof)
panic: http: multiple registrations for /debug/pprof/

goroutine 1 [running]:
net/http.(*ServeMux).Handle(0x2022de0, 0x1713f52, 0xd, 0x1fc3120, 0x1780750)
	/usr/local/go/src/net/http/server.go:2254 +0x610
net/http.(*ServeMux).HandleFunc(0x2022de0, 0x1713f52, 0xd, 0x1780750)
	/usr/local/go/src/net/http/server.go:2286 +0x55
net/http.HandleFunc(0x1713f52, 0xd, 0x1780750)
	/usr/local/go/src/net/http/server.go:2298 +0x4b
k8s.io/dns/cmd/kube-dns/app.(*KubeDNSServer).setupHandlers(0xc420014b40)
	/go/src/k8s.io/dns/cmd/kube-dns/app/server.go:147 +0x1a5
k8s.io/dns/cmd/kube-dns/app.(*KubeDNSServer).Run(0xc420014b40)
	/go/src/k8s.io/dns/cmd/kube-dns/app/server.go:119 +0x7b
main.main()
	/go/src/k8s.io/dns/cmd/kube-dns/dns.go:51 +0x2f8
```

The problem is introduced in #128.  we have imported "net/http/pprof" in cmd/kube-dns/app/options/options.go, which have already registered handlers for `/debug/pprof/`.

--- a/pkg/e2e/dns/dns.go
+++ b/pkg/e2e/dns/dns.go
@@ -59,8 +59,18 @@ func (kd *KubeDNS) Start(args ...string) {
 		kd.isRunning = false
 	}()
 
+	// dns service
 	om.Eventually(func() error {
 		conn, err := net.Dial("tcp", "localhost:10053")
+		if err == nil {
+			conn.Close()
+		}
+		return err
+	}).Should(om.Succeed())
+
+	// health check service
+	om.Eventually(func() error {
+		conn, err := net.Dial("tcp", "localhost:8081")
 		if err == nil {
 			conn.Close()
 		}

